README.md

```diff
@@ -0,0 +1,372 @@
+ID:          1de87g7c30g75qechgh7l5b2bha6r04e
+External ID: 34322be7-b2a7-45c2-af39-2c684ce624e1
+API URL:     https://api.rh-moa-test-cluster1.j9n4.s1.devshift.org:6443
+Console URL: https://console-openshift-console.apps.rh-moa-test-cluster1.j9n4.s1.devshift.org
+Nodes:       Master: 3, Infra: 3, Compute: 4
+Region:      us-east-2
+State:       ready
+Created:     May 27, 2020
+```
+
+Navigate to the `Console URL` and log in using your Github credentials.
+
+Once you are logged into your cluster, follow these steps to get your `oc` login command. In the top right of the OpenShift console, click your name and click **Copy Login Command**.  Click **github-1** and finally click **Display Token**. Copy and paste the `oc` login command into your terminal.
+
+```
+$ oc login --token=z3sgOGVDk0k4vbqo_wFqBQQTnT-nA-nQLb8XEmWnw4X --server=https://api.rh-moa-test-cluster1.j9n4.s1.devshift.org:6443
+Logged into "https://api.rh-moa-test-cluster1.j9n4.s1.devshift.org:6443" as "rh-moa-test-user" using the token provided.
+
+You have access to 67 projects, the list has been suppressed. You can list all projects with 'oc projects'
+
+Using project "default".
+```
+
+Run a simple `oc` command to verify everything is setup properly and you are logged in.
+
+```
+$ oc version
+Client Version: 4.4.0-202005231254-4a4cd75
+Server Version: 4.3.18
+Kubernetes Version: v1.16.2
+```
+
+Run the following command to verify your user only has dedicated-admin access. As a dedicated-admin you should receive some errors when running the following command:
+
+```
+$ oc get all -n openshift-apiserver
+NAME                  READY   STATUS    RESTARTS   AGE
+pod/apiserver-6ndg2   1/1     Running   0          17h
+pod/apiserver-lrmxs   1/1     Running   0          17h
+pod/apiserver-tsqhz   1/1     Running   0          17h
+NAME          TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+service/api   ClusterIP   172.30.23.241   <none>        443/TCP   17h
+NAME                       DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                     AGE
+daemonset.apps/apiserver   3         3         3       3            3           node-role.kubernetes.io/master=   17h
+Error from server (Forbidden): horizontalpodautoscalers.autoscaling is forbidden: User "rh-moa-test-user" cannot list resource "horizontalpodautoscalers" in API group "autoscaling" in the namespace 
+"openshift-apiserver"
+Error from server (Forbidden): jobs.batch is forbidden: User "rh-moa-test-user" cannot list resource "jobs" in API group "batch" in the namespace "openshift-apiserver"
+Error from server (Forbidden): cronjobs.batch is forbidden: User "rh-moa-test-user" cannot list resource "cronjobs" in API group "batch" in the namespace "openshift-apiserver"
+Error from server (Forbidden): imagestreams.image.openshift.io is forbidden: User "rh-moa-test-user" cannot list resource "imagestreams" in API group "image.openshift.io" in the namespace "openshift
+-apiserver"
+```
+
```

Comment:
I wonder if we push the cluster-admin stuff all the way to the end, and here we begin the addons example?